### PR TITLE
Add Compose ktlint rules

### PR DESCRIPTION
**Pull request by JakeWharton**

### CODE REVIEW

File: sample-viewmodel/src/main/java/com/example/molecule/viewmodel/presentationLogic.kt [hunk 1]
```diff
@@ -38,12 +38,12 @@ data class Model(
...
   @Composable
   override fun models(events: Flow<Event>): Model {
-    return PupperPicsPresenter(events, PupperPicsService())
+    return pupperPicsPresenter(events, PupperPicsService())
   }
 }
 
 @Composable
-fun PupperPicsPresenter(events: Flow<Event>, service: PupperPicsService): Model {
+fun pupperPicsPresenter(events: Flow<Event>, service: PupperPicsService): Model {
```
Comment (JakeWharton) [inline sample-viewmodel/src/main/java/com/example/molecule/viewmodel/presentationLogic.kt:46]: Should we change this naming convention with the casing change?

Comment (mattprecious) [reply sample-viewmodel/src/main/java/com/example/molecule/viewmodel/presentationLogic.kt:46]: Seems fine to me. @chris-horner?

**Review by mattprecious — APPROVED** [review]

File: sample-viewmodel/src/main/java/com/example/molecule/viewmodel/presentationLogic.kt (hunk 1 above)
Comment (jingibus) [reply sample-viewmodel/src/main/java/com/example/molecule/viewmodel/presentationLogic.kt:46]: 🤷 

I usually use lowercase when decomposing presenters in my own work. So: feels fine

**Review by jingibus — APPROVED** [review]

File: sample-viewmodel/src/main/java/com/example/molecule/viewmodel/presentationLogic.kt (hunk 1 above)
Comment (chris-horner) [reply sample-viewmodel/src/main/java/com/example/molecule/viewmodel/presentationLogic.kt:46]: All g with me 👍

**Pull request merged**